# HG changeset patch
# User Rik <rik@octave.org>
# Date 1536338858 25200
#      Fri Sep 07 09:47:38 2018 -0700
# Branch stable
# Node ID 60fd3c44769fc6706823cf7c78cee93b62d59afd
# Parent  1f52a96c8c21a552dd29fde1c63bd0c43c3ea99f
Fix BIST test in cset 1f52a96c8c21.

* data.cc (Fmod): Fix expected value for BIST test where dividend is negative
and divisor is 0.

diff --git a/libinterp/corefcn/data.cc b/libinterp/corefcn/data.cc
--- a/libinterp/corefcn/data.cc
+++ b/libinterp/corefcn/data.cc
@@ -902,17 +902,17 @@ negative numbers or when the values are 
 
 %!assert <*45587> (signbit (mod (-0, 0)))
 %!assert <*45587> (! signbit (mod (0, -0)))
 
 %!assert <*42627> (mod (0.94, 0.01), 0.0)
 
 %!assert <*54602> (mod (int8 (125), int8 (-25)), int8 (0))
 %!assert <*54602> (mod (int8 (-125), int8 (-25)), int8 (0))
-%!assert <*54602> (mod (int8 (-125), int8 (0)), int8 (0))
+%!assert <*54602> (mod (int8 (-125), int8 (0)), int8 (-125))
 %!assert <*54602> (mod (int8 (0), int8 (-25)), int8 (0))
 
 */
 
 
 #define DATA_REDUCTION(FCN)                                             \
                                                                         \
   int nargin = args.length ();                                          \
